htdocs/core/triggers/dolibarrtriggers.class.php

```diff
@@ -124,31 +143,29 @@ public function getDesc()
 	 *
 	 * @return string Version of trigger file
 	 */
-	public function getVersion()
+	public function getVersion(): string
```

Comment:
As the transition from PHP7.0 to PHP7.1+ is not yet complete by a lot of users, for the moment, we must avoid the use of the type in the function
"function(): type"
We will continue to wait a little bit that the nb of users with php 7.0 is lower (A lot of users first upgrade their Dolibarr to then upgrade the PHP according to Dolibarr requirement, so having a Dolibarr not hanging on fatal error with PHP 7.0 is still important to allow migration when this direction is used) .

Reply:
Thanks for explanation, I'll keep this in mind.

Reply:
For reference https://www.php.net/manual/en/migration70.new-features.php